roles/setup_server/tasks/main.yaml

```diff
@@ -124,6 +124,7 @@
 
 - name: Add source for custom packages
   when:
+    - hostvars[inventory_hostname].ansible_distribution != "Fedora"
```

Comment:
This PR resolves the need for this https://github.com/redhat-cop/infra.osbuild/pull/67